packages/block-library/src/loginout/index.php

```diff
@@ -16,6 +16,7 @@
  */
 function render_block_core_loginout( $attributes ) {
 
+	// This current url fetching logic matches with the core: https://github.com/WordPress/WordPress/blob/6612d90f6c8ee9e917dc2dfcbcc24e120a5746ea/wp-includes/general-template.php#L528
 	// Build the redirect URL.
 	$current_url = ( is_ssl() ? 'https://' : 'http://' ) . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
```

Comment:
```suggestion
	/*
	 * Build the redirect URL. This current url fetching logic matches with the core.
	 *
	 * @see https://github.com/WordPress/wordpress-develop/blob/6bf62e58d21739938f3bb3f9e16ba702baf9c2cc/src/wp-includes/general-template.php#L528.
	 */
	$current_url = ( is_ssl() ? 'https://' : 'http://' ) . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
```

- Let's use the PHPDoc format.
- It would be good to reference the development repo (`WordPress/wordpress-develop`), not the mirror repo (`WordPress/WordPress`),